recipes/libgcc/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: libgcc
+  version: 5.2.0
```

Comment:
Is this from gcc 5? As gcc 4 is used in a lot of cases too, I wonder if we should have both versions built somehow.